apisix/plugins/request-id.lua

```diff
@@ -214,6 +214,8 @@ function _M.rewrite(conf, ctx)
     local uuid_val = get_request_id(conf.algorithm)
     if not headers[conf.header_name] then
         core.request.set_header(ctx, conf.header_name, uuid_val)
+    else
+        uuid_val = headers[conf.header_name]
```

Comment:
I think we can avoid calling get_request_id if the id exists.

Reply:
This change should be submitted with an switch. In my situation, i can't trust the request id I received and must rewrite it with a new uuid.